modules/rest-api/pages/rbac.adoc

```diff
@@ -618,6 +619,130 @@ An attempt to add a user to one or more non-existent groups fails with `400 Bad
+
+.Example
+The following example backs up all users and groups. 
+----
+include::rest-api:example$rbac-backup-users-groups.sh[tag=backup-everything]  
+----
+
+The output from running the previous command on a database with one internal and one external user is:
+
+[source,json]
+----
+include::rest-api:example$rbac-backup-users-groups.sh[tag=backup-everything-output]  
+----
+
+IMPORTANT: The backup contains the hashed passwords for users defined in the local authentication domain. Be sure to keep the contents of the backup secure.
+
+.Parameters
+
+exclude=<filterExpression>::
+Excludes users and groups from the backup. The backup skips any user or group that matches the `filterExpression`.  The `filterExpression` can be one of the following:
++
+* `*`: excludes everything. 
+* `admin`: excludes the Full Administrator
+* `group:<groupWildcard>`: excludes group definitions whose name matches the `groupWildcard` wildcard expression. For example, `exclude=group:admin*` excludes any groups whose name starts with the string `admin`.
+* `user:<local|external|\*>:<userWildcard>`: excludes users. The `local`, `external`, or `*` expression sets the authentication domain to which the argument applies. The `userWildcard` wildcard expression determines which usernames the backup excludes. For example, `exclude=user:*:john*` excludes any user whose username starts with the string `john` in both the internal and external authentication domain.
+* `permission:<permissionExpression>`: excludes groups and users who have the permission matching `permissionExpression`. For example, `exclude=permission:cluster!backup_admin` excludes any user or group that has the backup admin permission at the database level.
+
+include=<filterExression>::
+Includes matching users and groups in the backup. Only users or groups that match the `filterExpression` are included. All users and groups that do not match the expression are excluded. The `filterExpression` can be one of the following:
++
+* `*`: includes everything. 
+* `admin`: includes only the Full Administrator
+* `group:<groupWildcard>`: includes only group definitions whose name matches the `groupWildcard` wildcard expression. For example, `include=group:analyst*` includes only groups whose name starts with the string `analyst`.
+* `user:<local|external|\*>:<userWildcard>`: includes users. The `local`, `external`, or `*` expression sets the authentication domain to which the argument applies. The `userWildcard` wildcard expression determines which usernames the backup includes. For example, `include=user:local:tom`  includes just the user named `tom` defined in the local authentication domain. 
+* `permission:<permissionExpression>`: includes groups and users who have the permission in the `permissionExpression`. For example, `include=permission:cluster.bucket[travel-sample]!data_writer` includes any user or group that has data writer permissions on the `travel-sample` bucket.
```

Comment:
I think `cluster.bucket[travel-sample]!data_writer` is not a correct permission.
data_writer is a role, not a permission.
If you mean to write a permission that has access to bucket docs, I think it makes sense to write something like the following:
`cluster.collection[travel-sample:*:*].data.docs!any`

Reply:
Updated the example.